### DISCUSSION TITLE
Hide collections in library

### DISCUSSION THREAD

**Discussion by aldfeu** [meisnate12/Plex-Meta-Manager#1239] [Q&A]
Is there a way to hide collection(s) from library. ALl collections I created (top ranking, genre + actors) are always displayed in collections tab + library tab.

So it is more or less not possible anymore to use the library tab because I need to scroll down a dozen of views before I can acces my movies.

I have already tried to set visible or no some contents, in the interface or in config.yml... none of them works.

Please help.

thanks

**Comment by chazlarson** [comment]
If you never want collections in the library view at all, turn them off in the library advanced settings.

If you want to hide only some, set the collection visibility as you desire.

If you are having trouble with the visibility settings, best would be to open a thread in the discord and include a log.

**Comment by aldfeu** [comment]
It works thanks.

Per collections items, I tried to hide/show... it did not worked. But this issue is already identified.
They remained displayed, whatever I select.
